flight/search/helper.py

```diff
@@ -0,0 +1,64 @@
+"""
+Summary
+-------
+Several useful functions repeatedly used in other files
+"""
+from math import sqrt
+
+AIR_DROP_AREA: list[tuple[float, float]] = [
+    (38.31442311312976, -76.54522971451763),
+    (38.31421041772561, -76.54400246436776),
+    (38.3144070396263, -76.54394394383165),
+    (38.31461622313521, -76.54516993186949),
+    (38.31442311312976, -76.54522971451763),
+]
+
+
+def calculate_dist(p_1: tuple[float, float], p_2: tuple[float, float]) -> float:
+    """
+    calculates the Euclidian distance between two points
+
+    Parameters
+    ----------
+    p_1: tuple[float, float]
+        the first point
+    p_2: tuple[float, float]
+        The second point
+
+    Returns
+    -------
+    distance: float
+        The Euclidian distance between these two points
+    """
+    return sqrt((p_1[0] - p_2[0]) ** 2 + (p_1[1] - p_2[1]) ** 2)
+
+
+def get_bounds(points: list[tuple[float, float]]) -> dict[str, list[float]]:
+    """
+    returns the vertices of the smallest square that encompasses all
+    the given points.
+
+    Parameters
+    ----------
+    points: list[tuple[float | float]]
+        the collection of points that define the given shape
+
+    Returns
+    -------
+    bounds: dict[str, list[float]]
+        The bounds of the search area
+    """
+    x_bounds: list[float] = [float("inf"), float("-inf")]
+    y_bounds: list[float] = [float("inf"), float("-inf")]
+
+    i: int
+    dim: tuple[int, list[float]]
+    for i, _ in enumerate(points):
+        for dim in ((0, x_bounds), (1, y_bounds)):
```

Comment:
`dim` here should be type hinted inside the first loop rather than outside of it.